python/ray/tests/test_metrics.py

```diff
@@ -148,7 +148,13 @@ def remote_store(self):
         try:
             webui_url = addresses["webui_url"]
             webui_url = webui_url.replace("localhost", "http://127.0.0.1")
-            raylet_info = requests.get(webui_url + "/api/raylet_info").json()
+            response = requests.get(webui_url + "/api/raylet_info")
+            response.raise_for_status()
+            try:
+                raylet_info = response.json()
+            except Exception as ex:
+                print("failed response: {}".format(response.text))
+                raise ex
```

Comment:
Is this change related?

Reply:
This change will print details when test fail, it is useful for debug.